packages/manager/modules/pci/src/projects/project/private-registry/private-registry.constants.js

```diff
@@ -0,0 +1,40 @@
+export const PRIVATE_REGISTRY_STATUS_MAP = {
+  ERROR: 'error',
+  READY: 'success',
+  DELETED: 'error',
+  SUSPENDED: 'error',
+  INSTALLING: 'info',
+  UPDATING: 'warning',
+  RESTORING: 'warning',
+  SUSPENDING: 'warning',
+  DELETING: 'warning',
+};
+
+export const PRIVATE_REGISTRY_STATUS = {
+  ERROR: 'ERROR',
+  READY: 'READY',
+  DELETED: 'DELETED',
+  SUSPENDED: 'SUSPENDED',
+  INSTALLING: 'INSTALLING',
+  UPDATING: 'UPDATING',
+  RESTORING: 'RESTORING',
+  SUSPENDING: 'SUSPENDING',
+  DELETING: 'DELETING',
+};
+
+export const GUIDES = [
+  {
+    id: 'configure',
+    link: 'https://docs.ovh.com/',
+  },
+  {
+    id: 'registry',
+    link: 'https://docs.ovh.com/',
+  },
+  {
+    id: 'deploy',
+    link: 'https://docs.ovh.com/',
+  },
+];
+
+export const DELETE_CONFIRMATION_INPUT = /^DELETE$/;
```

Comment:
There's no `export default `don't you get an error ?

Reply:
No error if you `import { DELETE_CONFIRMATION_INPUT } from './private-registry.constants.js'; `
It will throw an error if you `import constants from './private-registry.constants.js';`

Reply:
If I remember correctly you'll get an `eslint` error for not exporting something by default

Reply:
There is no error but i have changed this now